public_html/wp-content/plugins/wordcamp-payments/javascript/payment-requests.js

```diff
@@ -47,6 +45,18 @@ jQuery( document ).ready( function( $ ) {
 				$notes.toggle(state);
 				$notes.find('textarea').attr('required', state);
 			}).trigger('change');
+
+			$('#payment_receipt_country_iso3166').on('select2:select', function(e) {
```

Comment:
Is it possible to use a standard event like `change`? That would avoid the need to update this code if we ever disable `select2` or switch to a different dropdown. IIRC select2 will still fire the native event.

Reply:
Ah, yeah, you're right. `change` is supposed to be used here.

Reply:
Ah, yeah, you're right. `change` is supposed to be used here. Thank you!

Reply:
Ah, yeah, you're right. `change` is supposed to be used here. Thanks!